crates/saya/core/Cargo.toml

```diff
@@ -16,11 +16,12 @@ saya-provider.workspace = true
 
 anyhow.workspace = true
 async-trait.workspace = true
-cairo-proof-parser.workspace = true
+cairo-proof-parser = { git = "https://github.com/cartridge-gg/cairo-proof-parser", rev = "9e8922780f064d194b45caef36a72cc211f8348d" }
```

Comment:
This is something that could be merged into the version used by the workspace? Or we should keep the versioning separated here?

Reply:
fixed